pkg/cmdlinefilter/cmdlinefilter.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package cmdlinefilter provides filters to modify the bootloader kernel
+// parameters before booting.
+package cmdlinefilter
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"unicode"
+
+	"github.com/u-root/u-root/pkg/cmdline"
+)
+
+// Remove filter the given cmdline by removing fields.
+func Remove(cl string, fields []string) string {
```

Comment:
These are some prime candidates for unit tests -- I'd love to have some?

Reply:
Done